### ISSUE TITLE
Unfollow with webmention

### ISSUE THREAD

**Issue by datakurre** [snarfed/bridgy-fed#320]
I was able to add follow by posting page with follow verb to https://fed.brid.gy/webmention, but could not figure out how to unfollow. Am I correct that unfollow is not yet supported?

(Still learning about this project and indie web practices in general, but so far loving what I see here.)

**Comment by snarfed** [comment]
Thanks for the kind words! And yes, this isn't currently supported. I think you'd [delete](https://indieweb.org/deleted) (HTTP 410) your follow. That's the usual practice for other kinds of posts and interactions, eg #30, but I'm not entirely sure about follows. [Will ask on chat.](https://indieweb.org/discuss)

**Comment by snarfed** [comment]
Yeah seems like standard 410 delete: https://indieweb.org/unfollow#How

**Comment by snarfed** [comment]
Maybe! I generally try to follow existing microformats2 in the wild in Bridgy and Bridgy Fed, not invent or push new mf2. So if we start seeing `u-unfollow-of` in the wild on existing web sites, we could definitely consider it!

**Comment by snarfed** [comment]
Just FYI I've launched an unfollow UI: http://localhost:8080/docs#unfollow

**Comment by snarfed** [comment]
Status here: next step is probably to support HTTP 410 for unfollowing, unless we've recently started seeing a lot of `u-unfollow-posts` in the wild...

**Comment by snarfed** [comment]
Good chat in #indieweb-dev this morning: https://chat.indieweb.org/dev/2023-10-07#t1696693067602300
